specification/datalake-store/resource-manager/Microsoft.DataLakeStore/stable/2016-11-01/examples/Accounts_Create.json

```diff
@@ -169,6 +169,14 @@
           "firewallAllowAzureIps": "Enabled"
         }
       }
+    },
+    "409": {
```

Comment:
Please remove the status code "409" in all example files to pass the model validation check.

Reply:
thanks, made the change and validated locally that it does not complain. Since there is no way to validate this works for live validation, can @tiramisuq and @ruowan please take a look at my changes for adding ErrorResponse schema for error codes (default) and make sure my changes will indeed adress live validations. Here are the api issues related to invalid response code:


https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=75313DDC-5AF3-47FF-82F9-DAAAE21E4AA7
--

https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=8E8C7C18-967C-4349-AF11-7FAF2C5A9116
--

https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=F86FBE07-2947-4254-886D-633E6331E284
--

https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=779137A1-95DE-4B6D-BE8D-0F115A436A94
--


https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=779137A1-95DE-4B6D-BE8D-0F115A436A94
--